core/core.go

```diff
@@ -191,6 +191,10 @@ type Config struct {
 	// settable in browsers and cannot be set via environment variable. If
 	// provided, EthereumRPCURL will be ignored.
 	EthereumRPCClient ethclient.RPCClient `envvar:"-"`
+	// MaxBytesPerSecond is the maximum number of bytes per second that a peer is
+	// allowed to send before failing the bandwidth check. Defaults to 1 MiB, which
+	// is roughly 100x expected usage based on real world measurements.
```

Comment:
We should remove the "is roughly 100x expected usage based on real world measurements" since that no longer appears to be true.